### DISCUSSION TITLE
Failed to setup Minecraft, java.nio.file.ClosedFileSystemException: null

### DISCUSSION THREAD

**Discussion by Spacecake112** [FabricMC/fabric-api#3091] [Mod Dev Support] [answered]
I tried following a tutorial for making a 1.19 fabric mod, but now every time i try "load gradle changes" it spouts this error. Ive tried removing the gradle version manifest file in C:Users/Admin/.gradle/cache/fabric-loom but it didnt change anything.

![image](https://github.com/FabricMC/fabric/assets/95740140/36236e27-eb89-4439-a1c4-5d14181f4a6f)

If anyone needs anything else, just reply and ill try to provide what I can.

**Comment by Spacecake112** [comment, marked as answer]
This sounds so dumb but restarting Intellij IDEA fixed the issue
Welcome to coding ig

**Comment by TheK0tYaRa** [comment]
Welp, it doesn't help in VSCode.

**Comment by connorslade** [comment]
Deleting my `.gradle/loom-cache` fixed this for me.
